litmus-portal/frontend/src/components/WorkflowCard/styles.ts

```diff
@@ -191,7 +191,7 @@ const useStyles = makeStyles((theme) => ({
     flexDirection: 'row',
     alignItems: 'center',
     padding: theme.spacing(0.275, 0.75, 0.275, 0.75),
-    backgroundColor: theme.palette.secondary.light,
+    backgroundColor: theme.palette.secondary.dark,
```

Comment:
It is yellow as a warning in my hub and chaos hub @oumkale

Reply:
Do we have yellow theme?

Reply:
Yes @oumkale

Reply:
Check ‘theme.palette.warning.main’

Reply:
It's not exactly same. In hub it is #F6B92B and using theme #dba017. cc : @arkajyotiMukherjee

Reply:
You may use any yellow theme colour, litmus ui is not used in chaos hub currently @oumkale